src/TestRotations.java

```diff
@@ -0,0 +1,55 @@
+//: object/TestRotations.java
+
+import junit.framework.*;
+
+/**
+ * JUnit Class to test the rotation methods from the Quaternion and RotationMatrix classes.
+ *
```

Comment:
Or rotating the same vector with a quaternion and a matrix and check that the result is the same. My point is that when you write a unit test you basically want to see if the behavior you implemented is right. If you happen to have another object that does the same, but with a different implementation, it is a great candidate helper to write your test.